djstripe/mixins.py

```diff
@@ -40,9 +40,6 @@ def get_context_data(self, **kwargs):
         context = super(PaymentsContextMixin, self).get_context_data(**kwargs)
         context.update({
             "STRIPE_PUBLIC_KEY": settings.STRIPE_PUBLIC_KEY,
-            "PLAN_CHOICES": djstripe_settings.PLAN_CHOICES,
```

Comment:
PaymentsContextMixin should have some sort of Plan context. I think PLANS and PLAN_CHOICES (tupled version -- id, name) should be enough.